sub/dec_sub.c

```diff
@@ -369,6 +369,15 @@ char *sub_get_text(struct dec_sub *sub, double pts, enum sd_text_type type)
     return text;
 }
 
+char *sub_ass_get_extradata(struct dec_sub *sub)
+{
+    if (bstr_find0(bstr0(sub->sd->codec->codec), "ass") < 0)
```

Comment:
Why not just `strcmp(sub->sd->codec->codec, "ass")`?

Reply:
For some reason I thought it wouldn't be that simple without even trying... just doing strcmp works though, changed it.